### PR TITLE
adjust sidebar open state

### PR DESCRIPTION
**Pull request by klauskpm**
The sidebar starts closed, and slides when there is a selected track.

This closes issue #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -62,7 +62,9 @@ <h1>Playlist Curator</h1>
                     </ul>
                 </div>
             </main>
-            <aside>
+            oie
```
Comment (ScottHarrisonDev) [inline index.html]: I assume this is a typo? Works great other than this issue, if this is removed I'll accept the PR 👍

**Comment by klauskpm** [comment]
Hey @ScottHarrisonDev . The conflicts are now resolved!

**Review by ScottHarrisonDev — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (klauskpm) [reply index.html]: Yes, it is. Sorry about that. I'm changing it right away 👍

**Review by ScottHarrisonDev — APPROVED** [review]

**Comment by ScottHarrisonDev** [comment]
Great! Thank you!

**Comment by klauskpm** [comment]
Thank you :)

**Pull request merged**